Gemfile

```diff
@@ -11,7 +11,7 @@ gem "delayed_job", '~> 4'
 gem 'delayed_job_active_record'
 
 group :development, :test do
-  gem 'rspec-rails', '~> 2.0'
```

Comment:
not sure was it some special idea on holding rspec on ~> 2.0, but seems like it's better with newer versions